myrails/settings.py

```diff
@@ -75,16 +82,8 @@
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-    },
-    'OPTIONS': {
-
-        'timeout': 20,
```

Comment:
in the config file, you have a `connect_timeout` variable is that's okay?

Reply:
Yes, SQLite uses the `timeout` option while MySQL uses `connect_timeout`